### ISSUE TITLE
data process issue

### ISSUE THREAD

**Issue by superkevingit** [visionml/pytracking#98]
https://github.com/visionml/pytracking/blob/fbe1e501bc6c979c8cf05a78cac517db85d099e7/ltr/dataset/got10k.py#L67
Hi, when I use this code to train my own model, I found some errors in GOT-10k dataset, like image size change during a sequence(like "train_data/GOT-10k_Train_007947"), so I write a shell to detect these errors (find here https://github.com/hqucv/DatasetDebuger/blob/master/got10k_debuger.py), than I delete these bug sequences in "/ltr/data_specs/got10k_train_split.txt", but I find the index of this file is not corresponding to GOT-10k's indexes(start from GOT-10k_Train_000001), while the index of this file is start from 0.
So, I wonder how you generate the "got10k_train_split.txt" file or are there something wrong?

**Comment by goutamgmb** [comment]
Hi, we use the  'train/list.txt' file to load the sequences. The  "got10k_train_split.txt" contains the indices of the sequences in  the 'train/list.txt' file which are to be used for the training. Thus GOT-10k_Train_000001 gets the index 0, GOT-10k_Test_000002 gets index 1, and so on.

**Comment by superkevingit** [comment]
Thank you, I get it. But I still get three questions. First, there are some errors in GOT-10k dataset like image resolution change, I found these sequences still contains in train and validation data list files. But in some settings, the training code would be abnormal because of the error annotations and inconsistent picture sizes. Second, I found the validation data list of GOT-10k is not official validation dataset, why you choose these training data to be the validation data. Third, how you choose the sequence for different tasks, and what is the difference between VOT training dataset list and the common one.

**Comment by martin-danelljan** [comment]
1) Have you checked that the ground truth boxes are inconsistent for these sequences? Why would it be? It is natural that different sequences can have different resolutions. But we always sample a fixed-resolution crop, so this is not a problem for us.

2) We chose a *random* subset as our internal validation set. This is a standard good machine learning principle to follow. 

3) VOT employed GOT sequences for their new dataset, thus banning a subset of the data for training. We exclude those sequences in the 'vottrain' setting. Please train all our networks with this setting if you intend to evaluate on VOT in the future. (They seem to have removed this information now from the VOT website for some reason.)

**Comment by superkevingit** [comment]
Thank you for your patience. About the first question, I reinstalled part of the dataset and checked again, There are some errors in some sequences like GOT-10k_Train_007947, GOT-10k_Train_007948. The resolution of some images are not consistent to other frames and the content of  meta info file. This incorrectness would raise errors when I use inner-frames relationship in my training pipeline.

**Comment by martin-danelljan** [comment]
This relates to the GOT10k dataset. Please refer to their them about this issues. But thank you for pointing this out here as well. I would be happy if you can share your conclusions about these issues, especially if there turns out to be any issues with the annotations or meta data in GOT10k.

**Comment by superkevingit** [comment]
We remove the sequences that are not consistent in resolution at present using this [script](https://github.com/hqucv/DatasetDebuger). About the Dataset issue, we can refer to [this issue](https://github.com/got-10k/toolkit/issues/10#issuecomment-503367071) and just waiting for the  release. And thank you for your professionalism.

**Comment by martin-danelljan** [comment]
Thank you for this info :)
